dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/exceptions/i18n/SDOExceptionResource.nlsprops

```diff
@@ -0,0 +1,147 @@
+
+45036=A global property corresponding to the XML node being loaded was not found.
+
+45037=The prefix [{0}] is used but not declared in the XML schema.
+
+45038=Cannot perform operation on property [{0}] because it is not reachable from the path [{1}]. The path is invalid, or one or more Data Objects on the path are null.
+
+45039=An error occurred accessing the externalizableDelegator field [{0}] on the DataObject.
+
+45040=Cannot perform operation [{0}] with null parameter [{1}].
+
+45041=The value [{0}] of class [{1}] is invalid for property [{2}] of type [{3}].
+
+45100=An error occurred attempting to return the requested SDOHelperContext.  In an active WebLogic instance the application name is required for helper context cache lookups.  The application name could not be determined because {0} lookup failed.
+
+45101=An error occurred attempting to return the requested SDOHelperContext.  In an active WebLogic instance the application name is required for helper context cache lookups.  The application name could not be determined because {0} could not be invoked reflectively on {1}.
+
+45102=An error occurred attempting to return the requested SDOHelperContext.  In an active WebLogic instance the application name is required for helper context cache lookups.  The application name could not be determined because an ObjectName could not be created/returned for {0}.
+
+45103=An error occurred attempting to return the requested SDOHelperContext.  In an active WebLogic instance the application name is required for helper context cache lookups.  The application name could not be determined because an InitialContext could not be instantiated.
+
+45200=SDO/JAXB - An OXM descriptor could not be found that corresponds to the SDO type [{0}], ensure that a Java class is mapped to the XML type [{1}].
+
+45201=SDO/JAXB - An OXM mapping could not be found that corresponds to the SDO property [{0}], ensure that a Java property is mapped to the XML node [{1}].
+
+45202=SDO/JAXB - An SDO type could not be found that corresponds to the Java class [{0}].
+
+45203=SDO/JAXB - A schema reference must be set on the descriptor for the Java class [{0}].
+
+45204=SDO/JAXB - A schema context must be set on the schema reference on the descriptor for the Java class [{0}].
+
+45205=SDO/JAXB - An SDO type could not be found that corresponds to the Java class [{0}], ensure that an SDO type corresponds to the XML type [{1}].
+
+45206=SDO/JAXB - An error occurred creating a JAXB Unmarshaller.
+
+45207=An error occurred attempting to resolve a schema using the provided SchemaResolver.
+
+45208=Unable to map property {0} from type {1}. The javax.activation and javax.mail packages are required to map properties of type DataHandler. Ensure that both of these are available on the classpath.
+
+45209=An attempt was made to reset the ApplicationResolver instance on SDOHelperContext.  Only one set is permitted.
```

Comment:
The ApplicationResolver instance on the SDOHelperContext class could not be reset. Only one set is permitted.